DESCRIPTION

```diff
@@ -12,8 +12,13 @@ Date: 2016-08-16
 URLNote: https://github.com/krlmlr/
 BugReports: https://github.com/krlmlr/
 URL: http://krlmlr.github.io/
+Depends:
```

Comment:
Sorry, should have been `Imports`. It's actually because I've used some of `compat-purrr.R` which has an `rlang` dependency. I could of course use `vapply` etc. or have a `purrr` dependency, but I prefer the `purrr` syntax, and full `purrr` seems overkill here.